src/components/CreateOrUpdateSectionForm/CreateOrUpdateSectionForm.tsx

```diff
@@ -168,6 +171,7 @@ export const CreateOrUpdateSectionForm = ({
         setSchedulable(arg);
         setSearch('');
     };
+    const setVideoCallLink = (link: string) => setValue('videoCallLink', link);
```

Comment:
I'm not sure I understand what is happening here

Reply:
This function needed to control videoCallLink in [SectionScheduleCalendar](https://github.com/taskany-inc/hire/pull/622/files#diff-b5509b3f41dc4c72c051422583518d78c94d656aac493e495d7c2452836deb61), when section assigning in calendar